iceoryx2-cli/iox2-service/src/commands.rs

```diff
@@ -66,3 +71,54 @@ pub fn details(service_name: String, filter: OutputFilter, format: Format) -> Re
     }
     Ok(())
 }
+
+pub fn discovery(
+    rate: u64,
+    publish_events: bool,
+    max_subscribers: usize,
+    send_notifications: bool,
+    max_listeners: usize,
+    format: Format,
+) -> Result<()> {
+    let discovery_config = DiscoveryConfig {
+        publish_events,
+        max_subscribers,
+        send_notifications,
+        max_listeners,
+        sync_on_initialization: true,
+        include_internal: false,
+    };
+
+    let mut service =
+        DiscoveryService::<ipc::Service>::create(&discovery_config, Config::global_config())
+            .map_err(|e| anyhow::anyhow!("failed to create service discovery service: {:?}", e))?;
+
+    println!("=== Service Started (rate: {}ms) ===", rate);
```

Comment:
can you please use everywhere the `info!()` log interface instead of `println`

Reply:
@elfenpiff The point is to make the stdout parsable by scripts or other programs outside of the iceoryx2 ecosystem.

e.g. if the output is JSON, a bash script could just tail the output and react to the output.

Reply:
Makes absolutely sense - keep it as it is. But we need to document and test it somehow somewhere - not that I forget this in 2 months, change the output and screw up user scripts.

Reply:
I created a follow-up: https://github.com/eclipse-iceoryx/iceoryx2/issues/709